lib/perl/Genome/SoftwareResult/User.t

```diff
@@ -68,6 +72,7 @@ my $new_sr = $class->with_registered_users(
             callback => \&_newly_created_callback,
             %users_hash
         );
+UR::Context->commit();
```

Comment:
Oh, yeah, that makes sense.  It seems like it would be good if the test checked that they were unlocked after commit.  If you extracted a lock method that allowed arguments to be passed to `Genome::Sys->lock_resource` you could pass `max_try => 1`.